ext/bcmath/libbcmath/src/raise.c

```diff
@@ -42,15 +41,13 @@
 void bc_raise(bc_num num1, long exponent, bc_num *result, size_t scale)
 {
 	bc_num temp, power;
-	size_t rscale;
-	size_t pwrscale;
-	size_t calcscale;
+	size_t rscale, pwrscale, calcscale;
```

Comment:
I actually kinda prefer them to be on their own individual line with their type. Makes diff easier if one variable needs a change.